cogs/annual_handover_and_reset.py

```diff
@@ -305,7 +304,7 @@ async def increment_year_channels(self, ctx: TeXBotApplicationContext) -> None:
 
         await initial_message.edit(
             content=(
-                ":hourglass: Creating new \"first-years\" channel and setting permissions... "
+                ':hourglass: Creating new "first-years" channel and setting permissions... '
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes